<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="qSlicerWidget" name="Planning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="PlanningWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <attribute name="title">
       <string>Planning</string>
     </attribute>
     <widget class="QWidget" name="PlanningStep1">
      <attribute name="title">
       <string>Step 1: Skin Segmentation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_planning1"/>
     </widget>
     <widget class="QWidget" name="PlanningStep2">
      <attribute name="title">
       <string>Step 2: Target Segmentation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_planning2"/>
     </widget>
     <widget class="QWidget" name="PlanningStep3">
      <attribute name="title">
       <string>Step 3: Trajectory Planning</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_planning2"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
